### PR TITLE
vPbx admin validations in client portal

### PR DESCRIPTION
**Pull request by mmadariaga**
Removed fake vpbx admin validations

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: web/client/src/components/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -14,7 +13,12 @@ const Dashboard = (props: DashboardProps) => {
 
     return (
         <Grid container spacing={3} className={className}>
-            {EntityMap.map((routeMapBlock: RouteMapBlock, key: number) => {
+            {EntityMap.map((routeMapBlock, key: number) => {
+
+                if (! routeMapBlock.children.length) {
```
Comment (Kaian) [inline web/client/src/components/Dashboard/Dashboard.tsx:18]: extra space between ! and variable

File: web/client/src/entities/Ddi/Field/RouteType.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const { choices, readOnly } = props;
+
+    const modifiedProperty = {...property} as ScalarProperty;
+    delete modifiedProperty.component;
+
+    const enumValues = {
+        ...modifiedProperty.enum
+    };
+
+    if (!aboutMe?.pbx) {
```
Comment (Kaian) [inline web/client/src/entities/Ddi/Field/RouteType.tsx:32]: Is ok to call it _pbx_ but I try to use _vpbx_ term everywhere

File: web/client/src/entities/Ddi/Form.tsx [hunk 3]
```diff
@@ -10,12 +11,39 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+    const skip: Array<string> = [];
+    if (!aboutMe?.pbx) {
+        skip.push(...[
+            'user',
+            'ivr',
+            'huntGroup',
+            'conditionalRoute',
+            'conferenceRoom',
+            'queue',
+        ]);
```
Comment (Kaian) [inline web/client/src/entities/Ddi/Form.tsx:25]: Again, there are other vpbx endpoints like friend, conditionalRoute.
Fax again is present in residential and vpbx afaik

**Review by Kaian — CHANGES_REQUESTED** [review]

File: web/client/src/entities/Ddi/Field/RouteType.tsx (hunk 2 above)
Comment (Kaian) [reply web/client/src/entities/Ddi/Field/RouteType.tsx:32]: image

Comment (mmadariaga) [reply web/client/src/entities/Ddi/Field/RouteType.tsx:32]: That is API response attribute name, we could replace it everywhere on it's own PR

File: web/client/src/entities/Ddi/Form.tsx (hunk 3 above)
Comment (mmadariaga) [reply web/client/src/entities/Ddi/Form.tsx:25]: Again

File: web/client/src/entities/Ddi/Field/RouteType.tsx (hunk 2 above)
Comment (Kaian) [reply web/client/src/entities/Ddi/Field/RouteType.tsx:32]: :+1: :see_no_evil:

**Review by Kaian — APPROVED** [review]

**Pull request merged**